participants/notify.py

```diff
@@ -307,6 +307,32 @@ def __init__(self):
         self.email_store = None
         self.default_sender = None
         self.allowed_attachment_dirs = None
+        self.email_whitelist = None
+        self.email_blacklist = None
+
+    def filter_emails_by_domains(self, emails):
+        """Filter emails by domain name
+        :param emails: list of emails to filter
+        :return: list of filtered email
+        """
+        if not self.email_whitelist and not self.email_blacklist:
+            return emails
+
+        allowed_emails = []
+        for email in emails:
+            params = email.split('@')
```

Comment:
This is only checked if there is a black or whitelist
Although it's only a partial check I guess if it's in the code it should work 'properly'

Reply:
although it's just a side-effect check of splitting the domain... so.... maybe not worth doing "validation" here

Reply:
I would say "proper" email validation is separate issue, and this is ok for checking that there is the required domain part for the white/blacklisting.